src/translations/en.json

```diff
@@ -317,7 +317,7 @@
     "currentDocumentStatus": "Current {{docStatus}} document",
     "draft": "Draft",
     "draftSavedSuccessfully": "Draft saved successfully.",
-    "lastSavedAgo": "Last saved {{distance, relativetime(minutes)}}",
+    "lastSavedAgo": "Last saved {{distance}} ago",
```

Comment:
We need to make this change on every translations file. I normally use openAI to fill in where I am unsure.

Reply:
Thanks for the feedback. I'll be happy to make the changes and modify each of the translations.